frappe/custom/doctype/customize_form/customize_form.js

```diff
@@ -89,10 +89,6 @@ frappe.ui.form.on("Customize Form", {
 
 	setup_sortable: function (frm) {
```

Comment:
this name should probably change now. since everything is sortable. also don't think `grid.refresh()` is required anymore.